setup.py

```diff
@@ -20,6 +20,6 @@
         "pyOpenSSL >= 19.1.0"
     ],
     extras_require={
-        "tests": ["timeout-decorator"],
+        "tests": ["pytest", "timeout-decorator", "funcy>=1.14"],
```

Comment:
Let's add a stronger pytest requirement, will make our lives easier in the future :)